CombatCrittersSharp/rest/payloads/PackPayload.cs

```diff
@@ -11,18 +11,20 @@ public record PackContentsPayload(
     CardPayload[] cards
 );
 
-public record PackCreatorPayload(
-    PackCardSlotPayload slot,
-    int[] contents,
-    PackPayload pack_details
+
+public record PackCardSlotItem(
+    int rarity, // rarity id
+    int weight // weights for each rarity will be summed, 
+               //the proportion of the total will be the probablity of getting a card with that rarity missing rarities will assume a weight of 0
 );
 
 public record PackCardSlotPayload(
     PackCardSlotItem[] rarityWeights // array of weights for rarities in this slot
 );
 
-public record PackCardSlotItem(
-    int rarity, // rarity id
-    int weight // weights for each rarity will be summed, 
-               //the proportion of the total will be the probablity of getting a card with that rarity missing rarities will assume a weight of 0
-);
\ No newline at end of file
+public record PackCreatorPayload(
+    PackCardSlotPayload slot,
```

Comment:
this should be an array of PackCardSlotPayloads and the field should be named `slots`